scrapy/settings/default_settings.py

```diff
@@ -230,9 +230,12 @@
 STATS_DUMP = True
 
 STATSMAILER_RCPTS = []
-
-TEMPLATES_DIR = abspath(join(dirname(__file__), '..', 'templates'))
-
+TEMPLATES_DIR_BASE = [
```

Comment:
I noticed that both of this path are the same. We need to remove one, I purpose the second one since it requires an extra import.
After one is removed, we have to add a path to the project in the position 0. I already found a way of getting it, but it needs more changes. So I will let you do some research, maybe you can find a better option. :smile: